src/connection.ts

```diff
@@ -820,6 +825,9 @@ export class Connection extends EventEmitter {
    * @private
    */
   protected async startSendLoop () {
+    if (this.receiveOnly) {
```

Comment:
Seems like it might cause problems if this side never sends the amount it's willing to receive